kubernetes/distributed.yaml

```diff
@@ -1,73 +1,64 @@
 apiVersion: v1
-kind: Service
-metadata:
-  name: dscheduler
-  labels:
-    name: dscheduler
-spec:
-  ports:
-    - port: 8786
-      targetPort: 8786
-      name: dscheduler-tcp
-    - port: 9786
-      targetPort: 9786
-      name: dscheduler-http
-  selector:
-    name: dscheduler
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: dscheduler-status
-  labels:
-    name: dscheduler-status
-spec:
-  type: LoadBalancer
-  ports:
-    - port: 8787
-      targetPort: 8787
-  selector:
-    name: dscheduler
----
-apiVersion: v1
 kind: ReplicationController
 metadata:
   labels:
-    name: dscheduler
-  name: dscheduler
+    name: dask-scheduler
+    app: distributed
+  name: dask-scheduler
 spec:
   replicas: 1
   selector:
-    name: dscheduler
+    name: dask-scheduler
   template:
     metadata:
       labels:
-        name: dscheduler
+        name: dask-scheduler
     spec:
       containers:
-        - name: dscheduler
-          image: ogrisel/distributed:latest
-          args: ["start-dscheduler.sh"]
+        - name: dask-scheduler
+          image: mdurant/docker-distributed:renew
```

Comment:
I expect this docker container version to change in the future.  Would using version numbers or latest make more sense here than a name like `renew`?